apps/studio/src/components/CoreTabs.vue

```diff
@@ -671,13 +689,16 @@ export default Vue.extend({
   },
   beforeDestroy() {
     this.unregisterHandlers(this.rootBindings)
+    window.removeEventListener('keydown', this.handleKeyDown);
   },
+
   async mounted() {
     await this.$store.dispatch('tabs/load')
     if (!this.tabItems?.length) {
       this.createQuery()
     }
     this.registerHandlers(this.rootBindings)
+    window.addEventListener("keydown", this.handleKeyPress)
```

Comment:
Instead of doing this, could we add items to the `keymap` computed property? This binds keyboard shortcuts to Vue actions.

Reply:
I employed the computed 'keymap' prop, and during my exploration, I discovered that it relies on 'v-hokey.' Unfortunately, the library lacks a regex matching feature. Consequently, I had to manually specify 'alt+number' nine times to cover all possible scenarios